plugin/proxy/README.md

```diff
@@ -49,6 +50,9 @@ proxy FROM TO... {
   200-399, then that backend is marked healthy for double the healthcheck duration.  If it doesn't,
   it is marked as unhealthy and no requests are routed to it.  If this option is not provided then
   health checks are disabled.  The default duration is 4 seconds ("4s").
+* `report_health` enable the inclusion of this proxy bloc as part of the global healh of CoreDNS 
```

Comment:
It doesn't work like this. If you implement health.Healther the plugin's health is include in the global health.

Reply:
Well, yes but that mean that everyone that have already a corefile with 'health' will have an immediate impact at upgrade.

this option would permit:
1- to have a ramp-up at upgrade
2- finally to have a fine grain selecting which proxy bloc should be added in the overall health status.

I am open for rollback this part if it is not considered worth it. let me know.

Reply:
NOTE: I am using that flag also to workaround a weird timeout test in proxy.go. 
Can you provide your opinion there ? Thanks!

Reply:
timeout test should be root-caused (time permitting).

I don't think we should push more into proxy at this time and work on making forward a better proxy. Reporting health in forward is 2 to 5 lines as the "all upstreams are dead" is handled in the foward ServeDNS handler and not sprinkled across the protocol implementations.

And on third thought: I think doing health for proxies is a bad idea. You're tying the process health to some upstream entity that we don't control; that a global outage of all your proxies right there.

Reply:
I created an issue #1474 for FORWARD. 
Please have a look at proposition of implementation. 

I will need policy option on that plugin. (Issue is already opened).

Thanks!

Reply:
OOOPS .. I forgot an issue. 
I cannot witch to FORWARD without gRPC.!

Reply:
and also that "Dnstap" is supported ...